polling_stations/apps/file_uploads/templates/file_uploads/upload.html

```diff
@@ -121,12 +121,18 @@ <h2>Upload EMS export</h2>
                 xhr.open("POST", url, true);
                 xhr.setRequestHeader("Content-Type", "application/json");
                 xhr.setRequestHeader("X-CSRFToken", getCookie('csrftoken'));
+                var election_date;
+                if (document.querySelector('input[name="election_date"]:checked')) {
+                    election_date = document.querySelector('input[name="election_date"]:checked').value;
+                } else {
+                    election_date = document.querySelector('input[name="election_date"]').value
+                }
```

Comment:
@symroe Would be good to get a double check on this from you. It's meant to address the issue that when there's only one upcoming election we don't show radio buttons, so `document.querySelector('input[name="election_date"]:checked')` is `null`.

Reply:
I think this way is fine as is. The other options are:

1. It looks like just getting the value works, but I don't know about browser support. See https://jsfiddle.net/vjop5xtq/
2. We could just render a checked radio button in the HTML rather than a hidden input (but also hide the radio button in CSS). This would mean the JS don't have to work with both cases